tests/acceptance/io_services_tests.py

```diff
@@ -735,19 +735,6 @@ def start(self):
         self.check_cancel_connect(socket.AF_INET)
 
 
-class TestConnectSocketIPv6Success(SocketConnectorTestBase,
```

Comment:
Why remove IPV6 tests?

Reply:
To remove the need for `sudo` which would make builds a lot faster. This PR is more of an experiment than anything, because we can't use custom RabbitMQ config when the package is used without also using `sudo`